.github/workflows/build.yml

```diff
@@ -416,36 +416,42 @@ jobs:
 
     # MinGW uses JWASM to compile the assembly code - since that isn't available on github actions, this run is disabled
```

Comment:
Well this comment isn't needed anymore. Dont feel like it needs to be fixed, its a simple documentation change I can do after the PR is merged.

Reply:
Sounds good, I'll leave it to you.

Reply:
https://github.com/KhronosGroup/Vulkan-Loader/pull/1308